index.html

```diff
@@ -134,6 +134,8 @@ <h3 id="veyon"><a href="https://veyon.io">Veyon</a></h3>
       <h3 id="gnome-remote-desktop"><a href="https://wiki.gnome.org/Projects/Mutter/RemoteDesktop">GNOME Remote Desktop</a></h3>
       <p>Remote desktop and screen casting in Wayland.</p>
       <h2 id="libvncclient">LibVNCClient</h2>
+      <h3 id="AVNC"><a href="https://github.com/gujjwal00/avnc">AVNC</a></h3>
```

Comment:
Thanks! Please move to the end to keep the order of appearance.

Reply:
Ok, done.